### PR TITLE
Load EEG recorded on >1 underlying file

### PR DESCRIPTION
**Pull request by leondavis1**

### CODE REVIEW

**Comment by mivade** [comment]
This looks good to me, assuming it's ready.

**Comment by mivade** [comment]
Actually, @leondavis1, can you update the `EEGReader` docstring example to show that epochs can be 3-tuples?

File: cmlreaders/readers/eeg.py [hunk 1]
```diff
@@ -388,12 +388,14 @@ def as_timeseries(self,
...
                 data = self.rereference(data, contacts, scheme)
+                pairs = scheme[['contact_1', 'contact_2']].to_records().tolist()
 
             # TODO: tstart
             ts.append(
-                TimeSeries(data, sample_rate, epochs=epochs, contacts=contacts)
-            )
-            return TimeSeries.concatenate(ts)
+                TimeSeries(data, sample_rate, epochs=epochs,
+                           contacts=contacts if scheme is None else pairs)
```
Comment (mivade) [inline cmlreaders/readers/eeg.py]: The intent of contacts was to actually be contacts. I don't think it makes sense to set it to pairs.

Comment (leondavis1) [reply cmlreaders/readers/eeg.py]: Sure, but leaving it as contacts simply didn't work alongside rereferencing

**Pull request merged**